rootfs/Dockerfile

```diff
@@ -13,13 +14,14 @@ RUN apt-get update \
 		python-dev \
 		--no-install-recommends \
 	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 \
-	&& pip install --disable-pip-version-check --no-cache-dir docker-py==1.8.1 \
+	&& pip install --disable-pip-version-check --no-cache-dir git+https://github.com/docker/docker-py.git@9b63bed6a0b5185b043e85df8c49d86d2c048aa1 \
```

Comment:
?

I'm just talking about changing this line to use docker-py 1.9.0. Is there some background context I'm missing that's blocking us from upgrading straight to 1.9.0 which includes your patch?